storage/storage_tests/storage_tests.cpp

```diff
@@ -328,19 +328,19 @@ UNIT_TEST(StorageTest_Smoke)
 {
   Storage storage;
 
-  TIndex const usaGeorgiaIndex = storage.FindIndexByFile("USA_Georgia");
-  TEST(IsIndexValid(usaGeorgiaIndex), ());
-  CountryFile usaGeorgiaFile = storage.GetCountryFile(usaGeorgiaIndex);
-  TEST_EQUAL(usaGeorgiaFile.GetNameWithExt(MapOptions::Map), "USA_Georgia" DATA_FILE_EXTENSION,
-             ());
-
   TIndex const georgiaIndex = storage.FindIndexByFile("Georgia");
   TEST(IsIndexValid(georgiaIndex), ());
+  CountryFile usaGeorgiaFile = storage.GetCountryFile(georgiaIndex);
+  TEST_EQUAL(usaGeorgiaFile.GetNameWithExt(MapOptions::Map),
+             "Georgia" DATA_FILE_EXTENSION, ());
+
+  if (version::IsSingleMwm(storage.GetCurrentDataVersion()))
+    return; // Test below is on routing mwm. It is not relevant for single mwm data.
```

Comment:
Following code tests car routing files, and is not relevant for a single mwm case.